### PR TITLE
[VEN-2879] Deploy IL Markets and Pools to Base Sepolia and Mainnet

### PR DESCRIPTION
**Pull request by web3rover**
## Description

Resolves VEN

### CODE REVIEW

File: multisig/simulations/basesepolia/vip-004/abi/rateModel.json
Comment (chechu) [inline multisig/simulations/basesepolia/vip-004/abi/rateModel.json:1]: Unneeded

**Review by chechu — APPROVED** [review]

File: multisig/simulations/basesepolia/vip-004/abi/rateModel.json
Comment (chechu) [reply multisig/simulations/basesepolia/vip-004/abi/rateModel.json:1]: https://github.com/VenusProtocol/vips/pull/427/commits/129233186943256cabd394577c14cb5864075648

**Review by chechu — COMMENTED** [review]
Mainnet commands lgtm

**Review by GitGuru7 — APPROVED** [review]

**Review by kkirka — APPROVED** [review]
basemainnet commands, addresses and risk parameters LGTM

**Pull request merged**